hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/key/OMKeyCommitRequest.java

```diff
@@ -545,22 +545,34 @@ public static OMRequest blockCommitKeyWithBucketLayoutFromOldClient(
     return req;
   }
 
+  /**
+   * Validates key commit requests.
+   * We do not want to allow clients to perform hsync or lease recovery requests
+   * until the cluster has finalized the HBase support feature.
+   *
+   * @param req - the request to validate
+   * @param ctx - the validation context
+   * @return the validated request
+   * @throws OMException if the request is invalid
+   */
   @RequestFeatureValidator(
       conditions = ValidationCondition.CLUSTER_NEEDS_FINALIZATION,
       processingPhase = RequestProcessingPhase.PRE_PROCESS,
       requestType = Type.CommitKey
   )
-  public static OMRequest disallowHsync(
+  public static OMRequest disallowHBase(
```

Comment:
Am I missing something? I didn't find where this disallowHsync or disallowHBase is called.